kuma/users/views.py

```diff
@@ -289,6 +295,17 @@ def revision_by_distinct_doc(list_of_revisions):
     return [documents[doc_id] for doc_id in sorted(documents)]
 
 
+def ban_and_revert_notification(spammer, moderator, info):
+    subject = u'[MDN] %s has been banned by %s' % (spammer, moderator)
+    context = {'spammer': spammer,
+               'moderator': moderator}
+    context.update(info)
```

Comment:
Actually, it's a fairly standard pattern to have a constructed context be updated by kwargs (or whatever) passed in, so that it can be overridden by (perhaps unanticipated) callers, viz: https://github.com/django/django/blob/1.9.8/django/views/generic/detail.py#L102-L108